spyder/widgets/about.py

```diff
@@ -105,70 +161,136 @@ def __init__(self, parent):
+
+        self.label_community.setWordWrap(True)
+        self.label_community.setAlignment(Qt.AlignTop)
+        self.label_community.setOpenExternalLinks(True)
+        self.label_community.setTextInteractionFlags(Qt.TextBrowserInteraction)
+        self.label_community.setContentsMargins(15, 0, 25, 0)
+
+        self.label_legal.setWordWrap(True)
+        self.label_legal.setAlignment(Qt.AlignTop)
+        self.label_legal.setOpenExternalLinks(True)
+        self.label_legal.setTextInteractionFlags(Qt.TextBrowserInteraction)
+        self.label_legal.setContentsMargins(15, 0, 25, 0)
 
         icon_filename = "spyder_about"
         pixmap = QPixmap(get_image_path(icon_filename))
         self.label_pic = QLabel(self)
         self.label_pic.setPixmap(
-            pixmap.scaledToWidth(64, Qt.SmoothTransformation))
-        self.label_pic.setAlignment(Qt.AlignTop)
+            pixmap.scaledToWidth(100, Qt.SmoothTransformation))
+        self.label_pic.setAlignment(Qt.AlignBottom)
+        self.info = QLabel((
+                    """
+                    <div style='font-family: "{font_family}";
+                        font-size: {font_size}pt;
+                        font-weight: normal;
+                        '>
+                    <p>
+                    <b>Spyder IDE</b>
+                    <br>{spyder_ver}
+                    <br> {revision}
+                    <br> """).format(
+                spyder_ver=versions['spyder'],
+                revision=revlink,
+                font_family=font_family,
+                font_size=font_size))
+        self.info.setAlignment(Qt.AlignHCenter)
 
-        btn = QPushButton(_("Copy to clipboard"), )
+        btn = QPushButton(_("Copy version info"), )
         bbox = QDialogButtonBox(QDialogButtonBox.Ok)
+        bbox.setStyleSheet(
+           f"font-size: {buttons_font_size};"
+           f"padding: {buttons_padding}"
+         )
+        btn.setStyleSheet(
+           f"font-size: {buttons_font_size};"
+           f"padding: {buttons_padding}"
+         )
 
         # Widget setup
         self.setWindowIcon(ima.icon('MessageBoxInformation'))
         self.setModal(False)
 
         # Layout
-        tophlayout = QHBoxLayout()
-        tophlayout.addWidget(self.label_pic)
-        tophlayout.addWidget(self.label)
+        piclayout = QVBoxLayout()
+        piclayout.addWidget(self.label_pic)
+        piclayout.addWidget(self.info)
+        piclayout.setContentsMargins(20, 0, 15, 0)
+
+        scroll_overview = QScrollArea(self)
+        scroll_overview.setWidgetResizable(True)
+        scroll_overview.setWidget(self.label_overview)
+
+        scroll_community = QScrollArea(self)
+        scroll_community.setWidgetResizable(True)
+        scroll_community.setWidget(self.label_community)
+
+        scroll_legal = QScrollArea(self)
+        scroll_legal.setWidgetResizable(True)
+        scroll_legal.setWidget(self.label_legal)
+
+        self.tabs = QTabWidget()
```

Comment:
I think this should be

    self.tabs = QTabWidget(self)

so that the parent of that widget is the About dialog. Is that not possible for some reason?